### PR TITLE
tools: added a tool to calculate rosbag acc.

### PR DESCRIPTION
**Pull request by yifeijiang**

### CODE REVIEW

File: modules/tools/rosbag/stat_acc.py [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+import sys
+import os
+import datetime
+
+from rosbag.bag import Bag
+
+
+kChassisTopic = '/apollo/canbus/chassis'
+kLocalizationTopic = '/apollo/localization/pose'
```
Comment (Capri2014) [inline modules/tools/rosbag/stat_acc.py]: Looks like kLocalizationTopic and knano not needed?

File: modules/tools/rosbag/stat_acc.py [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+    def calculate(self, bag_file, log_file):
+        """Calculate and log acc."""
+        last_t = None
+        last_max_abs_acc = 0
+        last_speed = None
+        with Bag(bag_file, 'r') as bag:
+            for topic, msg, t in bag.read_messages(topics=[kChassisTopic]):
+                if last_t is not None:
+                    acc = (msg.speed_mps - last_speed) / (
```
Comment (Capri2014) [inline modules/tools/rosbag/stat_acc.py]: Also not sure how accurate you want the acceleration to be, Lincoln can parse acceleration in chassis_details directly from chassis, you can refer to https://github.com/ApolloAuto/apollo/blob/master/modules/canbus/proto/chassis_detail.proto for more details.

File: modules/tools/rosbag/stat_acc.py (hunk 1 above)
Comment (startcode) [reply modules/tools/rosbag/stat_acc.py]: recommend use modules/tools/common/message_manager.py

File: modules/tools/rosbag/stat_acc.py [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+
+def main():
+    """Main function."""
+    pgm_path = os.path.dirname(os.path.realpath(__file__))
+    now = datetime.datetime.now().strftime("%Y-%m-%d_%H.%M.%S")
+    log_file = open(pgm_path + "/" + now + ".txt", "w")
+
+    mc = AccCalculator()
+    for bag_file in sys.argv[1:]:
```
Comment (startcode) [inline modules/tools/rosbag/stat_acc.py]: recommend to use argparse

**Pull request closed without merging**